tutorials/basics/stream/stream1.py

```diff
@@ -13,7 +13,8 @@ class StreamTest(rfm.RegressionTest):
     valid_systems = ['*']
     valid_prog_environs = ['gnu']
     prebuild_cmds = [
-        'wget http://www.cs.virginia.edu/stream/FTP/Code/stream.c',
+        'wget https://raw.githubusercontent.com/jeffhammond/STREAM/'
+        'master/stream.c'
```

Comment:
I would put that in a single line and add a `noqa: E501` annotation, because it's less confusing to read.